src/components/Checkbox/index.tsx

```diff
@@ -165,11 +171,7 @@ Checkbox.propTypes = {
   size: PropTypes.number,
   typographyVariant: PropTypes.oneOf(typographyVariants),
   valid: PropTypes.bool,
-  value: PropTypes.oneOfType([
-    PropTypes.string,
-    PropTypes.bool,
-    PropTypes.number,
-  ]),
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
bool missing ?

Reply:
if I add bool: "Type 'Requireable<string | number | boolean>' is not assignable to type 'Validator<((string | number) & (string | number | readonly string[])) | null | undefined>'."

Pretty sure boolean values are always converted to string?

Reply:
You're right, CheckboxProps only accept a string | number :)